cmd/create.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"fmt"
+	"kool-dev/kool/cmd/presets"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+// KoolCreate holds handlers and functions to implement the preset command logic
+type KoolCreate struct {
+	DefaultKoolService
+	parser presets.Parser
+	KoolDocker
+	KoolPreset
+}
+
+func init() {
+	var (
+		create    = NewKoolCreate()
+		createCmd = NewCreateCommand(create)
+	)
+
+	rootCmd.AddCommand(createCmd)
+}
+
+// NewKoolCreate creates a new handler for create logic
+func NewKoolCreate() *KoolCreate {
+	return &KoolCreate{
+		*newDefaultKoolService(),
+		&presets.DefaultParser{Presets: presets.GetAll()},
```

Comment:
I guess would be good if we could do this in a lazy fashion... Right now for absolutely every single execution of `kool` it will have to call NewKoolCreate on the `init()` of this file, and allocate the memory for the presets maps - which basically undo a change we made a while ago - https://github.com/kool-dev/kool/commit/788fc6b993b3cc8e173b961f60868911ac22c948

Not sure how to make it lazy as it is now though, need to think 🤔

Reply:
@fabriciojs @leandroncbrito instead of adding the presets on `NewKoolCreate`, we can add the `LoadPresets` function and call it inside create command execution. The same can be done on preset/init